nextflow/modules/gene_coverage/process/align_reads.nf

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process ALIGN_READS {
+  tag "Aligning reads to ${metagenome.simpleName}"
+  label "python_cpus"
```

Comment:
Something to figure out.  If I remember correctly these use a lot of cpu but not a lot of ram and/or time. That means they don't really fit neatly in with nf-core's `process_low/medium/high` which dramatically increase all requested resources at once. 

It's poorly name because I originally made it for one of the python calls, but the `python_cpus` label was meant as a high-cpu, low-memory task. 

There's definitely more optimization to be done in assigning process resources